docs/reference/getting-started.asciidoc

```diff
@@ -368,7 +368,7 @@ And the response:
 health status index    uuid                   pri rep docs.count docs.deleted store.size pri.store.size
 yellow open   customer 95SQ4TSUT7mWBT7VNHH67A   5   1          0            0       260b           260b
 --------------------------------------------------
-// TESTRESPONSE[s/95SQ4TSUT7mWBT7VNHH67A/.+/ s/260b/\\d+b/ _cat]
+// TESTRESPONSE[s/95SQ4TSUT7mWBT7VNHH67A/.+/ s/260b/\\d+(\\.\\d)?k?b/ _cat]
```

Comment:
Ah, damn regex! This looks like its going to work for sth. like `1234b` and `1.1kb` but will not match if we get two or more digits after the comma at some point. At least according to http://www.regexplanet.com/advanced/java/index.html, `\\d+(\\.\\d)?k?b` doesn't match e.g. `1.123kb`. I _think_ `\\d+\\.?\\d+?k?b` might, and I don't thikn we need the group?

Reply:
indeed, the grouping can be removed. No need to support `.123`, because the formatting is cut after one decimal by the `ByteSizeValue` class

Reply:
👍